catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -123,6 +128,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		return super.onOptionsItemSelected(item);
 	}
 
+	public boolean isRecoveredFromCrash() {
+		return PreferenceManager.getDefaultSharedPreferences(this).getBoolean(RECOVERED_FROM_CRASH, false);
+	}
+
+	public boolean checkIfCrashRecoveryAndFinishActivity(final Activity context) {
```

Comment:
`context == this` because it's called like `checkIfCrashRecoveryAndFinishActivity(this);`
However, it still looks nicer with the parameter context. So just keep it.

Maybe you can come up with a better name for `checkIfCrashRecoveryAndFinishActivity`?

Btw, it's definitely a good idea to declare the context parameter as final, good job! :+1: